addons/binding/org.openhab.binding.pioneeravr/ESH-INF/thing/thing-types.xml

```diff
@@ -78,10 +156,10 @@
 	<channel-group-type id="displayInformationGroup">
 		<label>Display</label>
 		<channels>
-			<channel id="displayInformation" typeId="displayInformationChannel" />
+            <channel id="displayInformation" typeId="displayInformationChannel" />
```

Comment:
Could you re-apply the code formatter on this file?

Reply:
done